docs/src/api.md

```
@@ -64,30 +76,85 @@ pub enum Tag {
+    /// Minimum string length to include (default: 4)
+    pub min_length: usize,          // --min-len
+    /// Restrict to a specific encoding
+    pub encoding: Option<EncodingFilter>, // --enc
+    /// Only include strings with these tags (empty = no filter)
+    pub include_tags: Vec<Tag>,     // --only-tags
+    /// Exclude strings with these tags (empty = no filter)
+    pub exclude_tags: Vec<Tag>,     // --no-tags
+    /// Limit output to top N strings by score
+    pub top_n: Option<usize>,      // --top
+}
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)